firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -50,33 +80,45 @@ void moveDiagonal(int dirX, int dirY, int spacesX, int spacesY)
   digitalWrite(xMotor[1], dirX);
   digitalWrite(yMotor[1], dirY);
   
-  if (numStepsX = numStepsY) 
+  if (numStepsX == numStepsY) 
   {
     for(i = 0; i < numStepsX; i++)
     {
-    digitalWrite(xMotor[0], LOW);
-    digitalWrite(yMotor[0], LOW);
-    delay(1);
-    digitalWrite(xMotor[0], HIGH);
-    digitalWrite(yMotor[0], HIGH);
+      digitalWrite(xMotor[0], LOW);
+      digitalWrite(yMotor[0], LOW);
+      delay(1);
+      digitalWrite(xMotor[0], HIGH);
+      digitalWrite(yMotor[0], HIGH);
     }
   } 
 
-  else 
+  else if (numStepsY > numStepsX && (numStepsY / numStepsX) == 2)
   {
-    float dist = sqrt(numStepsX * numStepsX + numStepsY * numStepsY);
-    float delayX = dist / numStepsX;
-    float delayY = dist / numStepsY;
+    //can I call setScale here even thought it's declared after?
```

Comment:
From now on, ask any questions in the commit the same way I'm giving you feedback. That keeps the code free of unrelated comments. Also, if you make a comment please have a space after the //